### DISCUSSION TITLE
Keeping latest releases in place

### DISCUSSION THREAD

**Discussion by bansal21ankit** [zxing-cpp/zxing-cpp#579] [Q&A] [answered]
Currently the [releases](https://github.com/zxing-cpp/zxing-cpp/releases) section has 2.0.0 as the latest one which was from Jan 6. Can we instead keep the all the releases and keep them marked based on a major/minor/testing variants ?

I am asking this because I am working on something where it will be handy and error-free to just download the zxing_reader.js and zxing_reader.wasm files during the pipelining process instead of generating them using Emscripten.

**Comment by axxel** [comment, marked as answer]
I don't quite understand your suggestion as there are all the releases kept exactly at the link you provided. And if you are interested in the wasm artifacts, they are kept in the "Assets" section, e.g. [here](https://github.com/zxing-cpp/zxing-cpp/releases/download/v2.0.0/wasm-artifacts.zip)

You can also get the very latest wasm-artifacts for each build directly from the "Actions" page of the CI system, e.g. [here](https://github.com/zxing-cpp/zxing-cpp/actions/runs/5264438004#artifacts).
